doc/src/pair_hybrid.rst

```diff
@@ -37,36 +42,57 @@ Examples
-pair style is assigned to each pair of atom types.  With the
-*hybrid/overlay* style, one or more pair styles can be assigned to
-each pair of atom types.  The assignment of pair styles to type pairs
-is made via the :doc:`pair_coeff <pair_coeff>` command.
+The *hybrid*, *hybrid/overlay*, and *hybrid/scaled* styles enable the
+use of multiple pair styles in one simulation.  With the *hybrid* style,
+exactly one pair style is assigned to each pair of atom types.  With the
+*hybrid/overlay* and *hybrid/scaled* styles, one or more pair styles can
+be assigned to each pair of atom types.  The assignment of pair styles
+to type pairs is made via the :doc:`pair_coeff <pair_coeff>` command.
+The *hybrid/scaled* style differs from the *hybrid/overlay* style by
+requiring a factor for each pair style that is used to scale all
+forces, energies and stresses computed by each sub-style.  Because of
+the additional complexity, the *hybrid/scaled* style will have more
+overhead and thus will be a bit slower than *hybrid/overlay*.
 
 Here are two examples of hybrid simulations.  The *hybrid* style could
-be used for a simulation of a metal droplet on a LJ surface.  The
-metal atoms interact with each other via an *eam* potential, the
-surface atoms interact with each other via a *lj/cut* potential, and
-the metal/surface interaction is also computed via a *lj/cut*
-potential.  The *hybrid/overlay* style could be used as in the second
-example above, where multiple potentials are superposed in an additive
-fashion to compute the interaction between atoms.  In this example,
-using *lj/cut* and *coul/long* together gives the same result as if
-the *lj/cut/coul/long* potential were used by itself.  In this case,
-it would be more efficient to use the single combined potential, but
-in general any combination of pair potentials can be used together in
-to produce an interaction that is not encoded in any single pair_style
-file, e.g. adding Coulombic forces between granular particles.
+be used for a simulation of a metal droplet on a LJ surface.  The metal
+atoms interact with each other via an *eam* potential, the surface atoms
+interact with each other via a *lj/cut* potential, and the metal/surface
+interaction is also computed via a *lj/cut* potential.  The
+*hybrid/overlay* style could be used as in the second example above,
+where multiple potentials are superposed in an additive fashion to
+compute the interaction between atoms.  In this example, using *lj/cut*
+and *coul/long* together gives the same result as if the
+*lj/cut/coul/long* potential were used by itself.  In this case, it
+would be more efficient to use the single combined potential, but in
+general any combination of pair potentials can be used together in to
+produce an interaction that is not encoded in any single pair_style
+file, e.g. adding Coulombic forces between granular particles.  The
+*hybrid/scaled* style enables more complex combinations of pair styles
```

Comment:
Make this sentence start a new paragraph.

If h/scaled were used in place of h/overlay, then each sub-style contribution could be weighted, e.g. fractional ... or negative ...
Furthermore ... during a simulation, this enables smoothly switching ...